package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "muskie",
     "description": "Manta public-facing WebAPI",
-    "version": "2.0.0",
+    "version": "999.0.0",
```

Comment:
Note to self to not let this 999.0.0 version go by. :)

Reply:
Yep, bumped to 2.1.0, given we're changing the API (but within the scope of what we said we'd do for mantav2, so no major version bump)